### DISCUSSION TITLE
tomogram ctf model question

### DISCUSSION THREAD

**Discussion by Phaips** [SBC-Utrecht/pytom-match-pick#235] [Questions]
Hey!

I ran some TM and forgot to add the `--tomogram-ctf-model phase-flip` flag. So I ran it another time with the flag and compared the score maps. However, they looked identical. Now I am wondering what the flag is actually doing. I usually always run the `per-tilt-weighting` so I assume in this case even without the flag a phase flipped ctf template would be used?

```
pytom_match_template.py \
-v pos27.mrc \
-a 27.tlt \
--dose-accumulation 27_exposure.txt \
--defocus 27_defocus.txt \
-t tmpl.mrc \
-d tomo_27 \
-m mask.mrc \
--particle-diameter 140 \
-s 2 2 1 \
--voxel-size-angstrom 7.6 \
-r \
--rng-seed 69 \
-g 0 \
--amplitude-contrast 0.07 \
--spherical-aberration 2.7 \
--voltage 300 \
--z-axis-rotational-symmetry 4 \
--per-tilt-weighting \
--non-spherical-mask \
--tomogram-ctf-model phase-flip \  # with and without
```

Thanks!! :)

**Comment by McHaillet** [comment]
Hi phaips! 

Phase flipping only affects the phases of the CTF after the first zero crossing. Thus, it could depend on the defocus of the dataset. If the defocus is low, the first zero crossing might only appear after the resolution limit of the tomogram, which is 7.6 * 2 ~ 15 angstrom. For a defocus of 1.5 the first zero crossing is barely before Nyquist (I used this tool to check: https://ctfsimulation.streamlit.app/). However, I assume your defocus for a tilt-series is stronger than that. 

Alternatively, the resolution might just be limited after the first crossing due to a dense cell environment, some small misalignments between tilts, etc.

You can try running the command with the option: `--log debug`. This will write out the 3D-CTF and the convoluted template. You can check with and without `phase-flip` to see if the 3D-CTF is indeed different (and that there is not some bug)!

Best,
Marten

*edited for readability

**Comment by Phaips** [comment]
Hi Marten,

Thanks for the quick reply! Indeed it looks like this is the case. The scores and the extracted particles are (nearly) identical. However, from the `--log debug` the difference between the template PSF's are as expected!
Edit: the actual convolved template does not have a big difference (also not the FFT)

no phase-flip ![image](https://github.com/user-attachments/assets/b85d5d11-5714-4eed-97ea-d6ff9703eaf9)
with phase-flip ![image](https://github.com/user-attachments/assets/26646672-042e-4737-986a-7debfcc8142f)
